docs/running_psalm/command_line_usage.md

```diff
@@ -29,7 +29,11 @@ Currently, Shepherd tracks type coverage (the percentage of types Psalm can infe
 Psalm has a couple of command-line options that will result in faster builds:
 
 - `--threads=[n]` to run Psalm’s analysis in a number of threads
-- `--diff` which only checks files you’ve updated (and their dependents).
-- `--diff-methods` which remembers Psalm’s output when scanning particular methods.
+- `--diff` which only checks files you’ve updated since the last run (and their dependents).
+- `--diff-methods` which only checks methods you’ve updated since the last run (and their dependents).
```

Comment:
It would be good to add something about why these are optional and not always on. I haven't been using them up to now, but I don't actually know in what use case it's better to have them switched off.

Reply:
@muglug said he intends to enable those flags by default:
https://twitter.com/psalmphp/status/1220126747175325696

Reply:
Yes, but I don't know if there'll still be a need for some users to turn them off.

Reply:
We had them on at Vimeo, and ran into essentially impossible-to-reproduce bugs in CI when running with `--find-unused-code`. But our CI is different now and it should be much easier to reproduce issues, so I'll soon be turning it back on and monitoring for a while.